source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,6 +124,151 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
+B. Create a Data Encryption Key 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we use the MongoDB driver to create and store a data 
+encryption key, used by CSFLE to automatically encrypt and decrypt data on 
+document fields. The following configuration values are required by the client 
+to generate a new data encryption key:
+
+* The local master key or AWS KMS master key access settings.
+* A MongoDB connection string that authenticates to a running server.
+* The key vault database and collection name.
+
+Upon creating a data encryption key, the driver returns the ``id`` field of
+a document that stores information about the encrypted key including:
+
+* The data encryption key, encrypted by the master key.
+* The key management service provider data.
+
+In this section, we generate a single data encryption key from the local
```

Comment:
"In this section" is used two paragraphs above. Suggest rewording, but not a blocker.

Reply:
Thanks for identifying that, will reword.